src/OrcaSlicer.cpp

```diff
@@ -891,8 +891,6 @@ int CLI::run(int argc, char **argv)
     // instruct the window manager to fall back to X server mode.
     ::setenv("GDK_BACKEND", "x11", /* replace */ true);
 
-    ::setenv("WEBKIT_DISABLE_COMPOSITING_MODE", "1", /* replace */ false);
```

Comment:
Not sure if it's proper fix.
Some Linux distro require this env variable set to 1 to work.
https://github.com/search?q=repo%3ASoftFever%2FOrcaSlicer+WEBKIT_DISABLE_COMPOSITING_MODE&type=issues

Reply:
Would you prefer that this was a command line option?

With this present on Arch Linux the app simply crashes shortly after launch.

Reply:
Based on the parameter description, it should not cause the application to crash. All this feature does is disable hardware acceleration. It appears that there might be a graphics driver issue specific to certain Linux distributions.

I will do some due diligence.